guides/common/modules/proc_creating-a-host.adoc

```diff
@@ -4,6 +4,9 @@
 Use this procedure to create a host in {ProjectName}.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-host_{context}[].
 
+.Prerequisites
+* Review the provider-specific procedures for creating a host in the {ProvisioningDocURL}[Provisioning guide].
```

Comment:
We have attributes for titles: You can always substitute "URL" for "Title": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-titles.adoc#L26